### ISSUE TITLE
Feature: Sort the account list

### ISSUE THREAD

**Issue by xkre** [xkre/YesPojiQuota#2]
Having the ability to sort through the account list would be nice. Currently it is only sorted through position when the account is added.

Possible sorting options:
1. Account name
2. Quota balance left

**Comment by LauraLaureus** [comment]
I'm taking a look at this. I'll comment if I have any doubt :)

**Comment by LauraLaureus** [comment]
I have a quite simple approach, but I cannot test it because VS ask me to use windows store, do you know how to test it? any tutorial or so?

**Comment by xkre** [comment]
Have you enabled developer mode on your pc? This is an UWP app by the way.

**Comment by LauraLaureus** [comment]
Yes, I have only to click implement button to reload the keys and so on...  I still working 😄

**Comment by LauraLaureus** [comment]
Now, I have no data to check if the sort is working. How can I chek it?

**Comment by xkre** [comment]
you can change this line on the `ViewModelLocator` to implement `QuotaServiceMock` and use dummy data.
https://github.com/xkre/YesPojiQuota/blob/d03ade8e9b987569635597b9c44262014b004087/YesPojiQuota.Windows/ViewModels/ViewModelLocator.cs#L51

https://github.com/xkre/YesPojiQuota/blob/d03ade8e9b987569635597b9c44262014b004087/YesPojiQuota.Core/Services/QuotaServiceMock.cs#L11

**Comment by mohdizzudin** [comment]
still unsolved? need help?

**Comment by LauraLaureus** [comment]
I made a commit yesterday, but I couldn't test it.
